src/pivot-table/hooks/use-column-width.ts

```diff
@@ -102,7 +101,7 @@ export default function useColumnWidth(
         // Use the max width of all measures
         width = Math.max(
           ...qMeasureInfo.map(({ qFallbackTitle, columnWidth }) => {
-            const fitToContentWidth = measureTextForContent(qFallbackTitle) + TOTAL_CELL_PADDING;
+            const fitToContentWidth = measureTextForMeasureValue(qFallbackTitle) + TOTAL_CELL_PADDING;
```

Comment:
hmm looks like this was wrong before. This should measure the width of the text for the measure labels, so I guess this should be `measureTextForDimensionValue` now?

Reply:
Nice catch, will fix